pkg/ccl/storageccl/key_rewriter_test.go

```diff
@@ -113,15 +113,9 @@ func TestKeyRewriter(t *testing.T) {
 		desc.ID = oldID + 10
 		desc2 := sqlbase.DescriptorTable
 		desc2.ID += 10
-		newKr, err := MakeKeyRewriter([]roachpb.ImportRequest_TableRekey{
```

Comment:
I like keeping this as MakeKeyRewriterFromRekeys so that it is tested.

Reply:
It is still in the test above, so it does have coverage, but I'll switch this one over too.